README.md

```diff
@@ -31,22 +31,22 @@ What is ripple-lib used for? The applications on the list linked above use `ripp
 
 See also: [RippleAPI Beginners Guide](https://xrpl.org/get-started-with-rippleapi-for-javascript.html)
 
-In an existing project (with `package.json`), install `ripple-lib`:
+In an existing project (with `package.json`), install `xrpl.js`:
 ```
-$ npm install ripple-lib
+$ npm install xrpl
 ```
```

Comment:
These two seem out of sync -> is it install `xrpl.js` or install `xrpl`?

Reply:
it's `install xrpl`, but the package is named `xrpl.js` - the `.js` doesn't actually mean anything from a node point of view. 
see: https://web3js.readthedocs.io/en/v1.5.2/getting-started.html